tools/rosgraph/src/rosgraph/roslogging.py

```diff
@@ -40,6 +40,8 @@
 import logging
 import logging.config
 
+import yaml
```

Comment:
The package currently doesn't depend on yaml. Please add the additional dependency to the package manifest.